ns1/datadog_checks/ns1/check.py

```diff
@@ -10,6 +10,7 @@
 class Ns1Check(AgentCheck):
     NS1_CACHE_KEY = "ns1.cache.key"
     NS1_SERVICE_CHECK = "ns1.can_connect"
+    LOG_MSG_PREFIX = "NS1 API "
```

Comment:
```suggestion
    LOG_MSG_PREFIX = "NS1 API"
```
Delete the space here since the space is already applied later